backends/platform/sdl/win32/win32.cpp

```diff
@@ -398,106 +397,4 @@ AudioCDManager *OSystem_Win32::createAudioCDManager() {
 	return createWin32AudioCDManager();
 }
 
-char *OSystem_Win32::convertEncoding(const char* to, const char *from, const char *string, size_t length) {
-	char *newString = nullptr;
-	char *result = OSystem_SDL::convertEncoding(to, from, string, length);
-	if (result != nullptr)
-		return result;
-
-	bool swapFromEndian = false;
-#ifdef SCUMM_BIG_ENDIAN
-	if (Common::String(from).hasSuffixIgnoreCase("le"))
-		swapFromEndian = true;
-#else
-	if (Common::String(from).hasSuffixIgnoreCase("be"))
-		swapFromEndian = true;
-#endif
-	if (swapFromEndian) {
-		if (Common::String(from).hasPrefixIgnoreCase("utf-16")) {
-			newString = Common::Encoding::switchEndian(string, length, 16);
-			from = "utf-16";
-		}
-		else if (Common::String(from).hasPrefixIgnoreCase("utf-32")) {
-			newString = Common::Encoding::switchEndian(string, length, 32);
-			from = "utf-32";
-		}
-		else
-			return nullptr;
-		if (newString != nullptr)
-			string = newString;
-		else
-			return nullptr;
-	}
-	bool swapToEndian = false;
-#ifdef SCUMM_BIG_ENDIAN
-		if (Common::String(to).hasSuffixIgnoreCase("le"))
-			swapToEndian = true;
-#else
-		if (Common::String(to).hasSuffixIgnoreCase("be"))
-			swapToEndian = true;
-#endif
-	// UTF-32 is really important for us, because it is used for the
-	// transliteration in Common::Encoding and Win32 cannot convert it
-	Common::String tempString;
-	if (Common::String(from).hasPrefixIgnoreCase("utf-32")) {
-		Common::U32String UTF32Str((const uint32 *)string, length / 4);
-		tempString = Common::convertUtf32ToUtf8(UTF32Str);
-		string = tempString.c_str();
-		from = "utf-8";
-	}
-	if (Common::String(to).hasPrefixIgnoreCase("utf-32")) {
-		char *UTF8Str = Common::Encoding::convert("utf-8", from, string, length);
-		Common::U32String UTF32Str = Common::convertUtf8ToUtf32(UTF8Str);
-		free(UTF8Str);
-		if (swapToEndian) {
-			result = Common::Encoding::switchEndian((const char *) UTF32Str.c_str(),
-						(UTF32Str.size() + 1) * 4,
-						32);
-		} else {
-			result = (char *) malloc((UTF32Str.size() + 1) * 4);
-			memcpy(result, UTF32Str.c_str(), (UTF32Str.size() + 1) * 4);
-		}
-		if (newString != nullptr)
-			free(newString);
-		return result;
-	}
-
-	// Add ending zeros
-	WCHAR *tmpStr;
-	if (Common::String(from).hasPrefixIgnoreCase("utf-16")) {
-		// Allocate space for string and 2 ending zeros
-		tmpStr = (WCHAR *) calloc(sizeof(char), length + 2);
-		if (!tmpStr) {
-			if (newString != nullptr)
-				free(newString);
-			warning("Could not allocate memory for string conversion");
-			return nullptr;
-		}
-		memcpy(tmpStr, string, length);
-	} else {
-		tmpStr = Win32::ansiToUnicode(string, Win32::getCodePageId(from));
```

Comment:
As a result of this PR, ``Win32::getCodePageId()`` is no longer used, meaning that that function can be removed as well.